databuilder/models/table_column_usage.py

```diff
@@ -75,16 +72,14 @@ def create_next_node(self):
         except StopIteration:
             return None
 
-    def _create_next_node(self):
-        # type: () -> Iterator[Any]
+    def create_nodes(self):
+        # type: () -> List[Dict[str, Any]]
+        results = []
         for col_reader in self.col_readers:
-            if col_reader.column != '*':
-                raise NotImplementedError('Column is not supported yet {}'.format(col_reader))
-            yield {
-                NODE_LABEL: TableColumnUsage.USER_NODE_LABEL,
-                NODE_KEY: self._get_user_key(col_reader.user_email),
-                TableColumnUsage.USER_NODE_EMAIL: col_reader.user_email
-            }
+            if col_reader.column == '*':
+                # todo: make sure it won't override existing node
+                results.extend(User(email=col_reader.user_email).create_nodes())
```

Comment:
Could we just yield instead of generating all result list to reduce memory foot print? (It's already using quite amount of memory)